### DISCUSSION TITLE
Creating a Safe Maths module for Soroban smart contracts

### DISCUSSION THREAD

**Discussion by ghost** [stellar/sorobounty-spectacular#13] [General]
Here's my submission about how to create and use a **safe maths module** on a Soroban smart contract.

I've been interested in mathematical operations for implementing into a Soroban smart contract. I've learned a lot while playing around with modules and math functions, and despite of catching this event a bit late as well as still on basics about Soroban, I plan to continue working in this repository.

Repository: https://github.com/fernandodavidmartinez/soroban-safe-maths-mod
Tutorial: https://dev.to/fernandodavidmartinez/creating-a-soroban-safemath-module-4307

**Comment by briwylde08** [comment]
Hi there! Thank you so much for your Sorobounty Spectacular submission! We will review it and get back to you within a week. 🎉

Best,
Bri

**Comment by briwylde08** [comment]
Hi @fernandodavidmartinez! We have reviewed your submission. [🎉](https://emojipedia.org/party-popper/)

Given our guidelines, your submission may qualify for an award of $500 worth of XLM. To receive that award, please fill out [this form](https://stellarform.typeform.com/to/HBjEyCJt) completely and accurately.

Thanks again! Please let us know if you have any questions.

Best,
Bri

**Comment by briwylde08** [comment]
Thank you for providing the requested information! You’ve passed KYC and we’ve logged your award payment of $500 worth of XLM, which will be sent to the wallet you designated in the next week.

Thanks again and please let me know if you have any questions!

Best,
Bri
